### PR TITLE
[4633][ADD] purchase_report_internal_reference

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4633](https://www.quartile.co/web#id=4633&cids=3&model=project.task&view_type=form)

### CODE REVIEW

**Comment by AungKoKoLin1997** [comment]
@yostashiro @kanda999 I still didn't change the company name because we have to fix dotfiles for the author check. If not, pre-commit will be failed. I will follow up this with new task for all repos.

File: purchase_report_internal_reference/reports/purchase_order_templates.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//th[@name='th_description']" position="after">
+            <th name="th_internal_reference"><strong>Buyer Part No</strong></th>
```
Comment (yostashiro) [inline purchase_report_internal_reference/reports/purchase_order_templates.xml]:
```suggestion
            <th name="th_internal_reference"><strong>Buyer Part No.</strong></th>
```

File: purchase_report_internal_reference/reports/purchase_quotation_templates.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="report_purchasequotation_document_inherit"
+        inherit_id="purchase.report_purchasequotation_document"
+    >
+        <xpath expr="//th[@name='th_description']" position="after">
+            <th name="th_internal_reference"><strong>Buyer Part No</strong></th>
```
Comment (yostashiro) [inline purchase_report_internal_reference/reports/purchase_quotation_templates.xml]:
```suggestion
            <th name="th_internal_reference"><strong>Buyer Part No.</strong></th>
```

**Review by yostashiro — COMMENTED** [review]
Please add CONTEXT.md and explain that vendors sometimes need to know their clients' part number (to print on product labels, delivery slip, etc.) and standard Odoo doesn't support this when vendor product code is set for the product.

**Review by kanda999 — APPROVED** [review]
Functional and Code review.

**Review by yostashiro — APPROVED** [review]

**Pull request merged**